src/components/BannerCarousel/BannerCarousel.tsx

```diff
@@ -0,0 +1,88 @@
+"use client";
+
+import {
+  Carousel,
+  CarouselContent,
+  CarouselItem,
+  CarouselNext,
+  CarouselPrevious,
+  DotButton,
+} from "@/components/ui/carousel";
+import Autoplay from "embla-carousel-autoplay";
+import { IMainBanner } from "@/utils/interfaces";
+import { Icon } from "../Icon/Icon";
+import Image from "next/image";
+import { LinkButton } from "../LinkButton/LinkButton";
+
+const MainBanner = ({ content }: { content: { fields: IMainBanner }[] }) => {
+  return (
+    <Carousel
+      className="relative w-full"
+      opts={{
+        align: "start",
+        loop: true,
+      }}
+      plugins={[
+        Autoplay({
+          delay: 15000,
+          stopOnInteraction: false,
+        }),
+      ]}
+    >
+      <CarouselContent>
+        {content.map((item, index) => {
+          const { title, subtitle, image } = item.fields;
+
+          return (
+            <CarouselItem key={index} className="relative">
+              <div className="overflow-hidden relative flex justify-center items-center w-full lg:mt-4 min-h-[380px] lg:min-h-[510px]">
+                <Image
+                  className="absolute w-full min-h-[450px] h-full object-cover rounded-t-md md:rounded-l-md md:rounded-tr-none z-0"
+                  src={`https:${image.fields.file.url}`}
+                  alt=""
+                  width={1000}
+                  height={600}
+                />
+                <div className="absolute top-0 right-0 h-full w-[100%] bg-gradient-to-l from-black via-transparent to-transparent"></div>
+                <div className="absolute top-0 left-0 h-full w-[140%] bg-gradient-to-r from-black via-transparent to-transparent"></div>
+
+                <div className="flex w-full p-6 2xl:p-0 h-full items-end justify-between max-w-[1440px] z-10">
```

Comment:
```suggestion
                <div className="flex w-full p-6 2xl:p-0 md:px-20 h-full items-end justify-between max-w-[1440px] z-10">
```

This will solve the overlay from the text with the arrows on screens where the arrows appear